armi/settings/caseSettings.py

```diff
@@ -58,6 +58,10 @@ class Settings:
         :id: I_ARMI_SETTING0
         :implements: R_ARMI_SETTING
 
+    .. impl:: Define a case title to go with the settings.
```

Comment:
Can this `impl` tag go down on `caseTitle`?
https://github.com/terrapower/armi/blob/8a332c9c1f85c1b0f4152a4014043b68049f5135/armi/settings/caseSettings.py#L109-L114

Reply:
I put this here instead of there, because I couldn't decide if it should go on the setter or the getter or both.

You think the getter? Sure.

Reply:
Done.